vars/signArtifacts.groovy

```diff
@@ -138,7 +138,24 @@ void call(Map args = [:]) {
                """
                 }
         }
-        else {
+        else if (args.platform == 'mac') {
+            withCredentials([usernamePassword(credentialsId: "${GITHUB_BOT_TOKEN_NAME}", usernameVariable: 'GITHUB_USER', passwordVariable: 'GITHUB_TOKEN'),
+                string(credentialsId: 'jenkins-signer-mac-role', variable: 'SIGNER_MAC_ROLE'),
+                string(credentialsId: 'jenkins-signer-mac-external-id', variable: 'SIGNER_MAC_EXTERNAL_ID'),
+                string(credentialsId: 'jenkins-signer-mac-unsigned-bucket', variable: 'SIGNER_MAC_UNSIGNED_BUCKET'),
+                string(credentialsId: 'jenkins-signer-mac-signed-bucket', variable: 'SIGNER_MAC_SIGNED_BUCKET')]) {
+                sh """
+                   #!/bin/bash
+                   set +x
```

Comment:
Suggest adding `set -e` as well so it will exit if any errors.

Reply:
Just copied from whatever is for linux https://github.com/opensearch-project/opensearch-build-libraries/pull/233/files#diff-63b9bb4025ab21358c7d296b6204b847ecfffac44b6b249a2363824b30c61b1fL127-L129